content/2_Bronze/Simulation.mdx

```diff
@@ -214,39 +219,46 @@ int main()
 ```java
 import java.util.*;
 import java.io.*;
-import java.lang.Math.*;
 
-public class mixmilk
-{
-	public static Kattio io; // Refer to General - Input/Output
-	public static final int N = 3;
-	public static int['Simulation'] capacity = new int[N], milk = new int[N];
+public class MixMilk {
 	/*
-	N is the number of buckets (which happens to be a constant value of 3)
-	capacity[i] is the maximum capacity of bucket i
-	milk[i] is the current amount of milk in bucket i
-	*/
-	public static void pour(int i, int j)
-	{
-		int amt=Math.min(milk[i], capacity[j]-milk[j]);
-		// Amount of milk to pour is the minimum of the remaining milk in bucket i and the available capacity in bucket j
-		milk[i]-=amt;
-		milk[j]+=amt;
+	 * N is the number of buckets (which is 3)
+	 * capacity[i] is the maximum capacity of bucket i
+	 * milk[i] is the current amount of milk in bucket i
+	 */
+	static int N = 3;
+	static int[] capacity = new int[N];
+	static int[] milk = new int[N];
+
+	public static void pour(int i, int j) {
+		// Amount of milk to pour is the minimum of remaining milk in bucket i
+		// and the available capacity in bucket j
+		int amount = Math.min(milk[i], capacity[j] - milk[j]);
+		milk[i] -= amt;
+		milk[j] += amt;
```

Comment:
```suggestion
		int amount = Math.min(milk[i], capacity[j] - milk[j]);
		milk[i] -= amount;
		milk[j] += amount;
```
did you test this code??

Reply:
Oh shoot yeah I decided to change the variable right after I tested it and didn't catch that thanks